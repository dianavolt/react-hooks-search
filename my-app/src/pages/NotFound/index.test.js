import React from 'react'
import { shallow } from 'enzyme'

import NotFound from '.'

const render = (props) => shallow(<NotFound {...props}/>)

describe('NotFound component', () => {
    let test
    let props

    describe('without props', () => {
        beforeEach(() => {
            test = render(props)
        })
        
        it('should match snapshot', () => {
            expect(test).toMatchSnapshot()
        })
    })
})
