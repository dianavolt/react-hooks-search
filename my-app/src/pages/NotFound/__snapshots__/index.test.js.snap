// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component without props should match snapshot 1`] = `
<div>
  NotFound
</div>
`;
